src/abstractions/index.ts

```diff
@@ -4,6 +4,7 @@ export { AuditService } from './audit.service';
 export { AuthService } from './auth.service';
 export { CipherService } from './cipher.service';
 export { CollectionService } from './collection.service';
+export { CryptoFunctionService } from './cryptoFunction.service';
 export { CryptoService } from './crypto.service';
 export { EnvironmentService } from './environment.service';
 export { FolderService } from './folder.service';
```

Comment:
I have not been using these index barrels for a while now and would actually like to do away with them completely. They create issues with importing dependencies on certain platforms.

Reply:
Nice, I think that's for the best. I added a deprecation warning to index.ts instead.